src/index.html

```diff
@@ -4,10 +4,315 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <script type="text/javascript" src="scripts/main.js"></script>
   </head>
+
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__top-line top-line">
```

Comment:
```suggestion
      <div class="header__actions actions">
```
Not sure what does `top-line` can mean. Use something more understandable